wordpress-mobile/WordPress-iOS
# [EU/US Compliance] Use WordPress colors in compliance popover

Fixes #21151

## Description
Fixes an issue where the compliance popover in WordPress was using the Jetpack brand colors

## Screenshots

| Before | After - Light | After - Dark |
|--------|--------|--------|
| ![CleanShot 2023-07-20 at 11 18 08@2x](https://github.com/wordpress-mobile/WordPress-iOS/assets/25306722/5c2a7e3a-ff2e-4afb-ac76-965bfca0b112) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-07-20 at 18 27 29](https://github.com/wordpress-mobile/WordPress-iOS/assets/25306722/dd18bcc3-c3ed-4b97-bf6e-ffc915b92395) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-07-20 at 18 27 35](https://github.com/wordpress-mobile/WordPress-iOS/assets/25306722/974fc149-983a-45d5-a8a9-f6861c20ce00) |

## Testing Instructions
1. Fresh Install and run the WordPress app (Must be located in EU countries or VPN, alternatively, you can use [proxyman.io](https://proxyman.io/) to stub the `/geo` request).
2. Login
3. You should eventually land on "My Site"
4. ✅ Make sure the popup appears
5. ✅ Make sure the popover uses the WordPress colors
6. Repeat the same steps for the Jetpack app

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
